src/Microsoft.Management.UI.Internal/ShowCommand/ViewModel/ModuleViewModel.cs

```diff
@@ -442,14 +442,12 @@ private static bool MatchesEvenIfInPlural(string commandName, string filter)
 
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-macos`
Test-Connection.TraceRoute.TraceRoute works
```powershell
Expected exactly '10.79.1.108', but got 127.0.0.1.
at <ScriptBlock>, /Users/runner/runners/2.164.8/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 257
257:             $result[0].TargetAddress | Should -BeExactly $realAddress
```